.c9/project.settings

```diff
@@ -1,4 +1,7 @@
 {
+    "build": {
```

Comment:
this is wrong!

Reply:
Not sure about this ;( have you tried GO?

Reply:
I think it is really cool. @kchia